readme updates with the reimbursement procedure (#27)
bh2smith/cow-solver-rewards: README.md

## Proposed Reimbursement Work Flow

> The solver reimbursements are to be executed each Tuesday with the accounting period of the last 7 days. If the payout can not be done on Tuesdays, its okay to execute them later, but still the accounting period should always be from 00:00 between Monday and Tuesday to 00:00 between Monday and Tuesday

Reviewer: I was under the impression we were going to be doing Monday (beginning) to Sunday (end) (= Monday beginning), but this is fine too.

Author: First of March is a tuesday, hence we start with tuesdays. That is how I understood the proposal

In order to do the payout, run the following scripts:
```
source .env
rm -r out/

> python -m src.fetch.transfer_file --start '2022-MM-DD' --end '2022-MM-DD'

Reviewer: We should not have to specify both dates. We should adapt the script to accept either

- an integer (the calendar  week) or
- the start date a date string `YYYY-MM-DD`.

The rest should be handled internally. I will make a PR for this.

Author: Yes. I was thinking the same but wanted to proceed with it right now for the first payout.
If you can create the PR quickly that would be great

```
Here, the start should be the date of the Tuesday from last week and the end day should be the Tuesday from this week.
I.e. for the first payout, we would run:

> `python -m src.fetch.transfer_file --start '2022-03-01' --end '2022-03-08' `

Reviewer: If this is the example, then your description of monday-tuesday above is incorrect.

The results of this will be from beginning march 1 (a monday) to end of monday the next week (a total of 8 days). 

According to the description above, this should be 
```suggestion
`python -m src.fetch.transfer_file --start '2022-03-02' --end '2022-03-08'`
```

Reviewer: Sorry - forget that. For some reason I thought March 1 was a Monday!

Author: First of March is a Tuesday in my calendar

and for the next one:
`python -m src.fetch.transfer_file --start '2022-03-08' --end '2022-03-15'`

> Please double check that the payout is reasonable. That means the eth sent should be between 30-80 ETH, depending on gas prices from last week. Also the amount of cow send should reflect 100x the amount of batches. Reasonable numbers are around 300000-500000, that means 500-700 batches a day.

Reviewer: ```suggestion
Please double check that the payout is reasonable. That means the eth sent should be between 30-80 ETH, depending on gas prices from last week. Also the amount of cow send should reflect 100x the amount of batches. Reasonable COW totals are around 300000-500000, that means 500-700 batches a day.
```

Also, it might happen that the slippage of a solver is bigger than the ETH payout. In this case, please do not proceed with the payout, until the root cause is known. Feel free to reach out the project maintainers to do the investigation.

> Note that we must wait some time after the period has ended for some data to finalize (e.g. `dex.trades`

Reviewer: We do not directly make use of the `dex.trades` in our project.

```suggestion
Note that we must wait some time after the period has ended for some data to finalize (e.g. `prices.usd`
```

> , `ethereum.transactions` our event data, etc...). Hence, the scripts should not be executed around midnight on Tuesdays.

Reviewer: ```suggestion
, `ethereum.transactions` our event data, etc...). Hence, the scripts should not be executed immediately after the accounting period has ended.
```

> Once the transfer_file is created and you checked the results for reasonability, please create the multi-send transaction with the link provided in the console.

Reviewer: ```suggestion
After generating the transfer file and double-checking the results, please create the multi-send transaction with the link provided in the console.
```

> Let the team mates know about it the dev-multisig channel and make sure it will be executed. Preferably, you use the eth from the proposer account(eth:0xd8Ca5FE380b68171155C7069B8df166db28befdd) to execute the tx.

Reviewer: ```suggestion
Inform the team of this proposed transaction in the #dev-multisig Slack channel and follow through to ensure execution. It is preferred that the transaction be executed by the proposer account(eth:0xd8Ca5FE380b68171155C7069B8df166db28befdd).
```
